### DISCUSSION TITLE
Playlists saved in Windows or Unix path convention?

### DISCUSSION THREAD

**Discussion by touwys** [Borewit/listFix#192] [Q&A]
When saving a playlist, does listFix() save it in the Unix path convention (/), or in the alternative Windows one?

**Comment by Borewit** [comment]
If you run listFix() on Windows it uses the Windows path convention, on Unix systems it uses the Unix path convention.

There is no possibility to enforce one or the other.

**Comment by touwys** [comment]
> If you run listFix() on Windows it uses the Windows path convention, on Unix systems it uses the Unix path convention.
> 
> There is no possibility to enforce one or the other.

Thanks, the reason I asked, is that the Unix path option exists in my music library manager/player, as shown in the screenshot below:

•
![image](https://github.com/Borewit/listFix/assets/65730626/61831c5e-1428-4a29-a0c1-5fd27c7905b2)
•

**Reply by Borewit** [reply]
I am curious how it writes absolute Windows path in Unix path convention, as there is no drive letter in Unix path.

**Comment by touwys** [comment]
I do not have the faintest idea what it's all about, as I've never used the setting. If you wish to investigate, you could perhaps install the [portable version of the player](https://getmusicbee.com/downloads/), and play with the "Library" settings located under the programme preferences (Ctrl+O).

----------------------------------------

25 Jul 2023 17:27:18 Borewit ***@***.***>:
